src/metabase/api/dashboard.clj

```diff
@@ -791,7 +784,9 @@
     GET /api/dashboard/1/params/abc/values?def=100"
   [id param-key :as {:keys [query-params]}]
   (let [dashboard (api/read-check Dashboard id)]
-    (param-values dashboard param-key query-params)))
+    ;; If a user can read the dashboard, then they can lookup the chain-filter. This also works with sandboxing.
+    (binding [qp.perms/*user-can-read-dashboard* true]
```

Comment:
Maybe it'd make sense to move the read check + binding into `param-values` so that it's shared by both endpoints automatically?

Reply:
Actually, after making the change and looking at it, `param-values` is called from more places than just those 2. That's probably why I originally wrapped the bindings there.